gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -0,0 +1,82 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationStandby < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr standby register'.freeze
+
+    attr_accessor :appliance_pg_data, :standby_host
+
+    def initialize(hash = {})
+      self.appliance_pg_data = ENV.fetch('APPLIANCE_PG_DATA')
+      set_defaults
+      super
+    end
+
+    def set_defaults
+      self.standby_host = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+      self.host         = nil
+      self.database     = "vmdb_production"
+      self.username     = "root"
+      self.password     = nil
+      self.cluster_name = "replication_cluster"
+      self.node_number  = 1
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Replication Standby Server\n")
+      ask_for_standby_host
+      ask_for_cluster_name
+      ask_for_node_number
+      ask_for_database_credentials
+      confirm
+    end
+
+    def ask_for_standby_host
+      ask_for_ip_or_hostname("Standby Server hostname or IP address", standby_host) if standby_host.blank? || !local?
+    end
+
+    def activate
+      say("Configuring Replication Standby Server...")
+      confirm_no_database? &&
+        already_configured? &&
+        create_config_file(standby_host) &&
+        clone_standby_server &&
+        start_postgres &&
+        register_standby_server
+    end
+
+    def confirm_no_database?
+      return true if Dir.glob("#{appliance_pg_data}/*").empty?
+      say ""
+      say("Appliance database found under: #{appliance_pg_data}")
+      say("Replication standby server can not be configured if the database already exists")
+      say("Remove the existing database before configuring as a standby server")
+      false
+    end
+
+    def clone_standby_server
+      params = { :h  => host,
```

Comment:
This is wrong.
For cloning the host should be the host of the primary server. I'm not sure how this is working if the host that you are cloning from is the same as the host in repmgr.conf (which should be the local server's hostname).

I think you need to be asking for the `primary` server's hostname when configuring a standby. And also the same comment applies here as in https://github.com/ManageIQ/manageiq/pull/9585/files#r72783940 (we should also not assume the local IP address)